lighthouse-cli/test/cli/__snapshots__/index-test.js.snap

```diff
@@ -1063,6 +1063,9 @@ Object {
         Object {
           "path": "accessibility",
         },
+        Object {
+          "path": "external-resource-links",
```

Comment:
travis failure is just needing to update this snapshot

Reply:
done